slave-bob-ubuntu/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM docker.io/fhwendy/ubuntu-openjdk:precise
+
+MAINTAINER Adam Saleh <asaleh@redhat.com>
+
+ENV HOME=/home/jenkins
+
+USER root
+# Install Jenkins 
+
+RUN mkdir -p /home/jenkins && \
+    chown -R 1001:0 /home/jenkins && \
+    chmod -R g+w /home/jenkins && \
+    wget https://github.com/openshift/origin/releases/download/v1.3.0/openshift-origin-client-tools-v1.3.0-3ab7af3d097b57f933eccef684a714f2368804e7-linux-64bit.tar.gz && \
+    tar -xvzf openshift-origin-client-tools-v1.3.0-3ab7af3d097b57f933eccef684a714f2368804e7-linux-64bit.tar.gz && \
+    mv openshift*/oc /usr/local/bin && \
+    rm -rf openshift-origin-client-tools-*
```

Comment:
For runing openshift related commands from the jenkins pipeline-script.
It is in the base slave for centos.